firebase-installations-interop/src/main/java/com/google/firebase/installations/FirebaseInstallationsApi.java

```diff
@@ -24,14 +28,35 @@
  */
 public interface FirebaseInstallationsApi {
 
+  /** Specifies the refresh options to get a FIS AuthToken. */
+  @IntDef({FORCE_REFRESH, DO_NOT_FORCE_REFRESH})
```

Comment:
@ciarand  I added the IntDef as per your suggestion. I tried incorporating as much as I understood. I think, there is scope for improvement. PTAL to see if I am correctly using it :) Thank you.

Reply:
The @IntDef part is what I meant (although I think we should make it a bitmap), but the stateful setters aren't quite what I had in mind. Something like this was what I was thinking:

`getAuthToken(@AuthTokenOptions int authTokenOptions);`

Reply:
Updated. PTAL. Thank you.